src/main/java/com/redhat/jenkins/plugins/cachet/GlobalCachetConfiguration.java

```diff
@@ -39,6 +39,16 @@
 public final class GlobalCachetConfiguration extends GlobalConfiguration {
 
     private String cachetUrl;
+    private Boolean ignoreSSL;
```

Comment:
Shouldn't this have a default value of false somewhere?  Or maybe BooleanUtils does this?

Reply:
I use BooleanUtils on another spot to handle the default...want me to handle it on the get() too?